addons/portal_project/security/ir.model.access.csv

```diff
@@ -3,11 +3,11 @@ access_project,project,project.model_project_project,base.group_portal,1,0,0,0
 access_task,task,project.model_project_task,base.group_portal,1,0,0,0
```

Comment:
All rules related to portal / public can be moved directly into project, as the groups are now defined in base.